ftplugin/orgmode/liborgmode/headings.py

```diff
@@ -333,6 +336,19 @@ def init_checkbox(_c):
 
 		return self
 
+	def init_logbook(self):
+		heading_end = self.start + len(self) - 1
+		self.logbook.clear()
+		for i, line in enumerate(self.document._content[self.start + 1:heading_end + 1]):
+			line_date = _text2orgdate(line)
+			if line_date is not None:
```

Comment:
``if line_date:`` is enough or ``if line_date is None: continue`` with unindenting the current if block